tensorflow/lite/kernels/op_macros.h

```diff
@@ -15,23 +15,69 @@ limitations under the License.
 #ifndef TENSORFLOW_LITE_KERNELS_OP_MACROS_H_
 #define TENSORFLOW_LITE_KERNELS_OP_MACROS_H_
 
+// If we're on a platform without standard IO functions, fall back to a
+// non-portable function.
+#ifdef TF_LITE_MCU_DEBUG_LOG
+
```

Comment:
This change is a bit surprising, can you add some details about why this is needed? Is it because porting the LSTM op brought a requirement for new macros to be supported?

Reply:
lstm_eval.cc uses TF_LITE_ASSERT MACRO, so this is required